FrontEnd/package.json

```diff
@@ -17,6 +17,7 @@
     "react-router-dom": "^6.4.1",
     "react-router-hash-link": "^2.4.3",
     "react-scripts": "5.0.1",
+    "swr": "^2.2.2",
```

Comment:
Do you use it?

Reply:
Now it's not in my PR. Thanks for comment :)